README.rst

```diff
@@ -8,36 +8,53 @@
 Transformer Engine
 ==================
 
+`Quickstart <#examples>`_ | `Installation <#installation>`_ | `User Guide <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/index.html>`_ | `Examples <https://github.com/NVIDIA/TransformerEngine/tree/main/examples>`_ | `Model Support <#model-support>`_ | `Integrations <#integrations>`_ | `Release notes <https://docs.nvidia.com/deeplearning/transformer-engine/release-notes/index.html>`_
```

Comment:
1. I think having this as a bullet would be easier to read instead of in a single line?
2. Quickstart currently points to the readme examples but would it be better to point it to [this](https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/examples/quickstart.html)?

Reply:
1/ I prefer the navbar (used in JAX ecosystem) but I'm open to switching to bullet 
2/ I'd prefer quickstart to include code that someone can easily copy and run. But I'll update the copy in that section to also mention the guide for a deeper dive.

Reply:
Let's make it bold then, similar to JAX? It makes it easier to read